pkg/cnab/provider/action.go

```diff
@@ -140,9 +140,16 @@ func (r *Runtime) Execute(ctx context.Context, args ActionArguments) error {
 	currentRun.Bundle = b.Bundle
 	currentRun.BundleReference = args.BundleReference.Reference.String()
 	currentRun.BundleDigest = args.BundleReference.Digest.String()
-	currentRun.Parameters = args.Params
+
+	currentRun.Parameters.Parameters, err = r.sanitizer.CleanRawParameters(args.Params, cnab.ExtendedBundle{b.Bundle}, currentRun.ID)
+	if err != nil {
+		return err
+	}
+
+	currentRun.EncodeParameterOverrides()
```

Comment:
Can we move this method onto the sanitizer so that it's with the other sensitive data logic in one place?